examples/hybrid/TurbulenceConvectionUtils.jl

```diff
@@ -133,6 +133,8 @@ function init_tc!(Y, p, param_set, namelist)
     CC.Fields.bycolumn(axes(Y.c)) do colidx
         # `nothing` goes into State because OrdinaryDiffEq.jl owns tendencies.
         state = tc_column_state(Y, aux, nothing, colidx)
+        aux_gm = TC.center_aux_grid_mean(state)
+        @. p.ᶜp[colidx] = aux_gm.p
```

Comment:
This will (eventually) be reversed, correct? Also, I think this should be moved to after `compute_ref_state!`, where `aux_gm.p` is populated

Reply:
`in cases like Bomex`the logic will stay this way (the dycore will match the pressure given by the balanced reference state of the case). We would at some point have to make up our minds about the by-column vs. global code structure and unify these by populating `p.ᶜp[colidx]` based on the case

Reply:
Ok, makes sense